### PR TITLE
ui create and edit calendars

### PR DESCRIPTION
**Pull request by nhunght**

### CODE REVIEW

File: app/views/calendars/_form.html.erb [hunk 1]
```diff
@@ -1,13 +1,23 @@
...
+    </div>
+    <div class="form">
+      <%= f.text_area :description, required: true, class: "form-control" %>
+    </div>
   </div>
-  <div>
-    <%= f.submit t "calendar.save" %>
+
+  <div class="button button-create">
+    <%= f.submit t("calendar.save"), class: "rc-button rc-button-submit"  %>
```
Comment (vietbq) [inline app/views/calendars/_form.html.erb]: remove 1 space

File: app/views/calendars/_form.html.erb [hunk 2]
```diff
@@ -1,13 +1,21 @@
 <%= form_for [current_user, @calendar] do |f| %>
-  <div>
+  <div class="form-action">
+    <%= link_to root_path, title: t("events.buttons.go_back"),
+      "data-no-turbolink": true, class: "btn-go-back" do %>
+        <i class="go-back"></i>
```
Comment (vietbq) [inline app/views/calendars/_form.html.erb]: remote 1 tab

**Comment by daodat18plus** [comment]
ok

File: app/views/calendars/show.html.erb [hunk 3]
```diff
@@ -1,10 +1,34 @@
...
-<li>
-  <%= link_to t("calendars.delete_calendar"), user_calendar_path(current_user, @calendar),
-    method: :delete, data: {confirm: t("calendars.confirm_delete")} %>
-</li>
+<div id="calendar-form">
+  <%= form_for [current_user, @calendar] do |f| %>
+    <div class="form-action">
+      <%= link_to root_path, title: t("events.buttons.go_back"),
+        "data-no-turbolink": true, class: "btn-go-back" do %>
+          <i class="go-back"></i>
```
Comment (vietbq) [inline app/views/calendars/show.html.erb]: remote 1 tab

**Pull request merged**